test/Interop/Cxx/reference/Inputs/print-special-members.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+#include <cstdio>
+
+struct MoveOnly {
+    int id;
+    MoveOnly() : id(0) { printf("MoveOnly %d created\n", id); }
+    MoveOnly(const MoveOnly&) = delete;
+    MoveOnly(MoveOnly&& other) : id(other.id + 1) { printf("MoveOnly %d move-created\n", id); }
+    ~MoveOnly() { printf("MoveOnly %d destroyed\n", id); }
+};
+
+struct Copyable {
+    int id;
+    Copyable() : id(0) { printf("Copyable %d created\n", id); }
+    Copyable(const Copyable& other) : id(other.id + 1) { printf("Copyable %d copy-created\n", id); }
```

Comment:
Is/can this copy constructor invoked when we do something like:

```swift
let c0 = Copyable()
let c1 = Copyable(c0) // or should it be Copyable(other: c0)?
```

Reply:
No, we do not import the copy constructor as an initializer into Swift. Users cannot invoke them directly.